nomad/auth/auth.go

```diff
@@ -0,0 +1,492 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package auth
+
+import (
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"net"
+	"time"
+
+	metrics "github.com/armon/go-metrics"
```

Comment:
ultra nit: the package alias is redundant

Reply:
Oops, sure is! See, I really just copy-and-pasted this from the original code with the minimum changes! :grinning: